main.nf

```diff
@@ -67,30 +66,40 @@ def helpMessage() {
     Other options:
       --outdir                      The output directory where the results will be saved
       --email                       Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
-      --clusterOptions              Extra SLURM options, used in conjunction with Uppmax.config
       -name                         Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic
       --seqCenter                   Text about sequencing center which will be added in the header of output bam files
       --protocol                    Library preparation protocol. Default: "illumina". Can be set as "illumina", "nextflex", "qiaseq" or "cats"
       --three_prime_adapter         3’ Adapter to trim. Default: None
       --skip_qc                     Skip all QC steps aside from MultiQC
```

Comment:
Ive started to be consistent with boolean parameters specified in the format `--skipQC` and parameters that require a value as `--skip_qc`. We havent started to enforce this but it would be nice if we could get some consistency. For example, `--seqCenter` would then be `--seq_center`

Reply:
Should `--length` be `--min_length` instead?

Reply:
Can simplify validation of files:
https://github.com/nf-core/smrnaseq/blob/6257c01c2db67834a5694d63dd708a7e18f487ac/main.nf#L150

With the following syntax:
https://github.com/nf-core/chipseq/blob/a76bd6981f5001840368079264d8157e54d068aa/main.nf#L147

Reply:
I changed the ones I saw very similar.